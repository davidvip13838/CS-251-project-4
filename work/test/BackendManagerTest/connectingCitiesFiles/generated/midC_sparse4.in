500
2176 6931
5946 3375
8595 6085
8660 1548
8569 2172
2579 7544
7498 5364
6662 1231
7014 1075
6932 1716
5311 1261
2076 2041
4539 1081
3897 5802
8455 3056
7636 4801
7292 9765
9173 3678
5253 9150
8755 624
9679 9728
6039 1775
8958 3953
8857 1444
1510 785
197 6303
3734 6724
4477 2520
7459 5271
6388 1545
1091 6721
4001 7281
94 4485
7749 2882
5255 1987
2199 4536
300 8215
9514 4550
355 5234
8531 1221
6318 6009
252 3736
9958 7232
6422 7509
9670 9473
3802 7895
4526 4895
321 2995
2125 5386
1019 3293
1614 2449
431 4361
2515 7297
219 6428
1528 6211
294 7464
33 6162
8770 4832
5780 7857
8574 8123
3128 5008
7423 1925
8418 5751
8045 870
7093 6801
2252 581
652 5471
2902 3532
3184 3585
3315 7743
3903 7009
418 3132
798 6917
3007 379
8358 9689
1179 2740
302 2731
5778 6747
8555 256
939 5969
672 9453
4652 2739
6608 1989
1091 6828
7260 3655
9480 1450
2068 8099
7927 4729
2186 2276
8582 9230
9673 6939
1828 357
7188 3321
2757 7534
1403 5012
7205 460
7710 2746
6716 1540
6698 53
4832 2527
5497 4246
1069 6437
5529 1017
4054 6756
3193 5495
3326 7354
2223 9544
4038 8977
6493 3677
9781 8601
9278 3837
7506 300
7999 6189
8100 3918
2063 128
16 7485
9580 4789
7054 7630
6272 636
765 4586
8876 8609
8662 5019
9893 9055
5365 4332
5000 1032
5687 7542
6369 6908
4069 6961
7763 8396
3368 765
6574 2462
1775 783
4366 280
9476 3018
3302 8312
5290 7284
2292 3233
8277 2373
8629 3977
5592 8490
8354 7798
6784 3673
323 7855
3546 9357
6834 7781
4916 9449
9674 4735
1198 3046
7229 4867
9597 8861
9855 478
9762 9023
3084 7380
2070 6419
1298 7906
4575 7774
5223 9441
9429 1486
8279 5037
4053 6220
7733 4267
2195 4841
2757 7144
5663 8775
8781 4402
8223 2410
1006 1480
4951 8936
2199 1110
1578 6469
6892 7724
8939 8051
6833 9541
4284 5746
9055 7940
5017 7485
5776 7373
180 9231
6716 102
1619 7113
6033 8395
8 5496
742 4922
5228 7947
4977 683
4137 2364
4628 759
3728 8357
8603 3180
2541 4138
1800 6817
5936 3947
6119 5357
9918 4520
5455 8573
1911 4163
8696 5019
8762 9404
6143 2210
7476 8661
9285 3182
9807 4455
9148 410
3245 2478
7855 8924
6686 2352
3396 1112
4319 1158
7732 2114
2669 8769
4214 8056
2724 7329
1575 9721
3818 1272
6051 5690
3477 1856
4310 7649
8770 6372
369 6400
6070 4384
3603 4324
8436 6118
2903 9004
7785 7241
4507 4023
7289 5812
2763 2034
9307 8196
3383 9884
8395 6884
1553 4451
3375 8595
2062 2578
1244 3513
4245 4788
2495 9533
2266 7692
8549 6127
4450 5064
8654 6166
697 417
6624 5771
394 2702
1544 8230
7635 6004
7843 9084
6074 3025
5669 3494
7079 7812
7412 1184
4090 6172
7308 1362
8473 3730
954 9354
7930 4761
2543 4514
1769 8959
9819 8973
8748 3863
3338 1786
7422 9743
7377 4304
2548 3694
6623 1559
3461 4877
914 2547
3617 4506
4750 2517
1492 183
2065 9193
7988 4389
7813 1997
627 9615
5051 2513
170 111
5007 606
3903 3852
3652 7863
6052 362
1008 4007
9921 6382
4193 8878
9425 3279
6188 5315
6449 8070
4887 8023
2599 5866
3214 9909
9626 8637
915 3877
2559 2928
9847 6485
5149 3807
4188 6585
8342 5599
9303 528
6154 8854
9013 8622
5691 5692
2627 4220
7108 5936
8117 4536
3158 8342
9881 609
6861 701
8955 4671
75 3888
4181 4566
8816 3117
8807 9937
389 7720
3177 9226
5094 6818
5708 2163
5985 2980
7250 3351
3363 863
9655 5485
147 9161
7582 1093
8804 6121
6024 7925
9838 6819
4729 8943
5576 5769
4330 2653
3559 6112
8577 2460
634 3847
1681 5653
1693 3270
2840 1700
8003 7831
895 7176
6996 2637
908 1494
3547 178
855 8279
2697 9979
6075 802
2422 7235
5648 5883
3753 7617
30 4705
2839 5541
7038 8729
8502 3640
1865 1612
5019 8559
9377 2259
758 7499
7027 1199
8908 8317
3868 7770
6978 8430
9484 3578
2715 3988
2538 1482
1727 4634
984 863
388 6381
4477 2090
7661 2823
7726 2958
4452 9610
568 1328
7305 162
9009 6172
9657 2881
3311 9642
4396 111
6506 1294
1531 4595
6518 2288
1244 6062
8803 1830
4549 5365
1883 8446
4686 4654
6231 7896
6654 652
3044 9442
489 6902
8523 632
6495 8907
1758 2641
1673 187
6634 1602
7554 926
2601 1865
5059 9838
8229 4749
593 8198
2071 5739
4939 4720
8558 2772
8161 509
6024 9062
4521 3706
2404 8887
3052 571
3839 8514
4733 3889
471 3718
2073 831
7412 5502
8890 1017
6569 3134
9577 8238
8929 6735
433 2226
6515 7289
8912 194
2493 2127
7159 7025
2267 7938
875 9057
3969 6752
8419 5224
2482 9661
7574 3062
9911 4939
1686 3556
5188 8026
8025 4636
2378 2828
3707 5306
754 445
9090 8448
7549 8874
546 7025
4878 711
9473 1509
3542 1113
3152 7647
6203 9747
835 2773
3443 5255
4249 376
1035 2977
5115 4514
526 7244
600 6515
5868 9076
6924 4487
4215 9525
2988 5585
6206 3605
3816 5715
1900 3857
4563 1469
3827 4247
8797 7182
3817 2428
6460 7297
8968 7492
8656 9716
3379 9336
9602 6768
4610 9101
919 3972
8744 690
8322 7649
4 9520
6295 4087
3029 8548
3273 4647
4047 6056
9322 7134
4582 1184
6796 4094
9235 4049
7571 7510
4195 3968
1897 4394
1857 3035
2789 1978
9421 1074
2944 9115
1665 6581
2944 9243
9110 6790
8112 9315
1294 2382
399 5705
3591 7869
6061 8777
3225 664
3559 1086
4323 7784
6099 203
2056 4540
391 6664
4730 1404
1584 1495
4965 9620
356 6349
9286 6840
9516 4153
6328 4833
