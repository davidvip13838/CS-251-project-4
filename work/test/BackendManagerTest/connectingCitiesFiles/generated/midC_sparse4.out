146541.27701015628